### DISCUSSION TITLE
3.3 argument and parameter definitions

### DISCUSSION THREAD

**Discussion by jdonwells** [TEALSK12/introduction-to-computer-science#348] [Ideas]
Should we sync our usage of argument and parameter with the Collage Board (AP) definitions?

**Comment by aspiece** [comment]
@jdonwells I am incorporating this into the PR I am working on right now. Thank you for bringing this to our attention. You are amazing for sharing what you find. It's great getting this insight directly from the field.

**Comment by jdonwells** [comment]
Here is what I am using for 3.3. https://docs.google.com/presentation/d/14Kjb7sdaD4Op6INZQvCwxkWXfyoH4uMThSgal4d673A/edit#slide=id.p1
